crowdsec-docs/unversioned/console/cti/cti_api_keys.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: CTI API Keys
+description: What can you access in the CTI API
+---
+
+import AcademyPromo from '@site/src/components/AcademyPromo';
+import ThemedImage from "@theme/ThemedImage";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+This section will describe how to access to our CTI API and the different plans available.   
+If you want to know more about the CTI API itself, you can check the [CTI API documentation](/u/cti_api/getting_started).
+
+# Creating a CTI API Key
+
+You can create a CTI API key via the** + ** short cut on the top right corner that will lead you to the CrowdSec Console settings, CTI API Key section.
+
+![CTI API Key Creation](/img/console/cti/cti_api_create_key.png)
+
+<ThemedImage
+    alt="CrowdSec Create API Key Page"
+    sources={{
+        light: useBaseUrl("/img/console_create_api_key_page_light.png"),
+        dark: useBaseUrl("/img/console_create_api_key_page_dark.png"),
+    }}
+/>
+
+# CTI Plans
+
+The CTI API contains multiple endpoints that are not all accessible with any plan.   
```

Comment:
"The CTI API features several endpoints, but not all of them are accessible with every plan."